app/clients/statusclient.js

```diff
@@ -31,10 +31,18 @@ export default class StatusClient {
 		return this._doGet("/api/incidents");
 	}
 
+	getIncidentsByType(type) {
+		return this._doGet("/api/incidents?type=" + type);
```

Comment:
Does the axios library have a nicer way to deal with query string params? Maybe a pain to get at when it's hidden behind _doGet though?